test/oracle.test.ts

```diff
@@ -0,0 +1,84 @@
+import {ethers} from 'ethers'
```

Comment:
where do you use it?

Reply:
Yeah I installed it and didn't use it, was playing with it to mimic `keccak256(abi.encode())` hahsing as web3js is not great at that... I removed it for now.